noxfile.py

```diff
@@ -87,11 +87,10 @@ def docs(session: nox.Session) -> None:
 
 @nox.session
 def build(session: nox.Session) -> None:
-    session.install("build", "check-manifest>=0.44", "twine")
+    session.install("build", "twine")
 
-    session.run("check-manifest", "--verbose")
     # If no argument is passed, build builds an sdist and then a wheel from
     # that sdist.
     session.run("python", "-m", "build")
 
-    session.run("twine", "check", "dist/*")
+    session.run("twine", "check", "--strict", "dist/*")
```

Comment:
is this the replacement for `check-manifest` ?

Reply:
No, it's just a stricter check for the package (I think including README). We don't have a manifest anymore.

Reply:
thanks for the info.